core/utils/collection.php

```diff
@@ -118,6 +118,18 @@ public function map( callable $callback ) {
 		return new static( array_combine( $keys, $items ) );
 	}
 
+	/**
+	 * Run a callback over each of the items.
+	 *
+	 * @param callable $callback
+	 * @return void
+	 */
+	public function each( callable $callback ) {
+		foreach ( $this->items as $key => $value ) {
+			$callback( $value, $key );
+		}
```

Comment:
Please allow chaining like `$collection->each(...)->map(...)`
```suggestion
		}
	return new static( $result );
```

Reply:
I think we need to return $this.
@nahmana check what they do on Laravel

Reply:
If it's not changed, it should return `$this`

Reply:
@nahmana please add in the description, that the method is not designed to change the array (with a reference to the `map` method)

Reply:
Done.